src/graphql/search/filters/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Context } from '../../config'
+import getCmsFilters from '../cms/filters'
+import { DCAT_ENDPOINTS } from '../datasets/config'
+import getDatasetsFilters from '../datasets/filters'
+import { combineFilters } from './utils'
+
+export default async (_: any, {}, { loaders }: Context) => {
+  try {
+    // Get the drupal theme taxonomy from the DataLoader
+    const cmsThemeTaxonomy: { value: any } = await loaders.cms.load(
+      `${process.env.CMS_URL}/jsonapi/taxonomy_term/themes`,
+    )
+
+    const [cmsThemeFilters] = getCmsFilters(cmsThemeTaxonomy.value) // And construct the filters
+
+    // Get the openapi taxonomy from the DataLoader
+    const openApiTaxonomy: any = await loaders.datasets.load(DCAT_ENDPOINTS['openapi'])
+    const [datasetsThemeFilters] = getDatasetsFilters(openApiTaxonomy.value)
+
+    return combineFilters([cmsThemeFilters, datasetsThemeFilters])
+  } catch (e) {
+    return e
```

Comment:
The callee of this method know how to deal with the return value of an error?

Reply:
The way GraphQL works is when there is an Error for a GraphQL type, it will return `null`. Unless it's a non-nullable field, which is not the case for the "base" types

Reply:
Ok, good to know, thanks!